src/Wrapper/Product.php

```diff
@@ -260,6 +283,27 @@ protected function processVariations(bool $checkAvailability = true): int
             $variationsProcessed++;
         }
 
+        if (!empty($variationsPriceData)) {
+            $priceColumn = array_column($variationsPriceData, self::PRICE);
+            array_multisort($priceColumn, SORT_ASC, $variationsPriceData);
+            $cheapestVariationData = reset($variationsPriceData);
+            $this->cheapestVariationId = $cheapestVariationData[self::VARIATION_ID];
+
+            if (!$hasImage && !empty($cheapestVariationData) && $cheapestVariationData[self::IMAGE]) {
+                $this->item->addImage($cheapestVariationData[self::IMAGE]);
+                $hasImage = true;
+            }
+
+            if (!$hasPrice && $cheapestVariationData[self::PRICE]) {
+                $this->item->addPrice($cheapestVariationData[self::PRICE]);
+                $hasPrice = true;
+            }
+        }
+
+        if (!$hasImage && $defaultImg) {
```

Comment:
Please don't use abbreviations.
```suggestion
        if (!$hasImage && $defaultImage) {
```